nebari/cli/main.py

```diff
@@ -50,6 +50,34 @@
 DEV_COMMAND_MSG = "Development tools and advanced features."
 
 
+def path_callback(value: Path) -> Path:
+    return value.expanduser().resolve()
+
+
+def config_path_callback(value: Path) -> Path:
+    value = path_callback(value)
+    if not value.is_file():
+        raise ValueError(f"Passed configuration path {value} does not exist!")
+    return value
+
+
+CONFIG_PATH_OPTION: Path = typer.Option(
```

Comment:
This is the only refactor I've made. Since the configuration file and output path is used by multiple subcommands, I've defined them once at the top. This avoids duplication in the code and makes it easier to maintain.